src/Service/CloudFormation/src/CloudFormationClient.php

```diff
@@ -176,16 +158,22 @@ protected function getEndpointMetadata(?string $region): array
 
     protected function getServiceCode(): string
```

Comment:
This function can be removed in all clients but the one tagged 1.0

Reply:
thanks, done